risc0/zkvm/methods/guest/Cargo.toml

```diff
@@ -34,3 +34,6 @@ debug = 1
 
 [package.metadata.release]
 release = false
+
+[features]
+redis = []
```

Comment:
why do we need this?

Reply:
we don't, but my rust-analyzer kept complaining that it doesn't have a redis feature and would stop working

Reply:
I [removed it](https://github.com/risc0/risc0/pull/2407/commits/5e5938602154f54233d77b4cfd0557569c23e43b) and now rust-analyzer isn't complaining anymore 🤷🏽